### PR TITLE
Update gdal-warp-bindings v1.1.1

### PR DESCRIPTION
**Pull request by CloudNiner**
# Overview

Updates gdal-warp-bindings to the newly released 1.1.1 to address MacOS dylib issues.

## Checklist

- [x] [./CHANGELOG.md](https://github.com/locationtech/geotrellis/blob/master/CHANGELOG.md) updated, if necessary. Link to the issue if closed, otherwise the PR.
- [x] GDALWarp bindings 1.1.1 [CQ #22803](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22803)

### CODE REVIEW

**Review by pomadchin — APPROVED** [review]
Thanks, works wonderful! Tests passed for me with the local brew GDAL 3.2.0 installed and symlinked to match the GDAL 3.1.x dylib version: `ln -s /usr/local/lib/libgdal.28.dylib /usr/local/lib/libgdal.27.dylib`

I left a minor comment about the CHANGELOG. And we need to file a CQ before merging it!

**Comment by CloudNiner** [comment]
CQ Approved!

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -Fix `LayoutTileSource` buffer should only be 1/2 a cellsize to avoid going out of bounds and creating `NODATA` values [#3302](https://github.com/locationtech/geotrellis/pull/3302)
 - Remove unused allocation from CroppedTile [#3297](https://github.com/locationtech/geotrellis/pull/3297)
 
+### Fixed
```
[the remark this answers is not in the document]
Comment (CloudNiner) [reply CHANGELOG.md]: Changed works for consistency. Updated.

**Comment by CloudNiner** [comment]
Going to await testing by @moradology before merge.

**Comment by pomadchin** [comment]
I merged in https://github.com/locationtech/geotrellis/pull/3295 and it created CHANGELOG conflicts. I resolved them though the UI and that is what 02839d8 about.

**Comment by CloudNiner** [comment]
👍

**Comment by pomadchin** [comment]
How about merging it in and rising a new issue if it would happen to be the case? We'd anyway have to publish a GDAL 3.2.x release soonish. I'd like to get this PR into the upcoming release.

cc @moradology @CloudNiner @notthatbreezy

**Pull request merged**